colpali_engine/trainer/contrastive_trainer.py

```diff
@@ -10,22 +10,11 @@ def __init__(self, loss_func, is_vision_model, *args, **kwargs):
 
     def compute_loss(self, model, inputs, return_outputs=False):
         query_outputs = model(input_ids=inputs["query_input_ids"], attention_mask=inputs["query_attention_mask"])
-        if self.is_vision_model:
```

Comment:
If the `is_vision_model` attribute is not used, can you put a comment next to `is_vision_model` to say that it's unused and will be removed in one of the next breaking version? (no need to remove it now, we're not in a hurry). I think we can also default it to `True` if that's the case.

Reply:
Yeah we dropped support for the other models so they are all vision models now (and it actually doesn't matter anymore since we just pass all relevant kwargs)

Reply:
Good catch